ansys/dpf/core/plotter.py

```diff
@@ -131,7 +131,8 @@ def add_field(self, field, meshed_region=None, show_max=False, show_min=False,
                   label_text_size=30, label_point_size=20, **kwargs):
         # Get the field name
         name = field.name.split("_")[0]
-        kwargs.setdefault("stitle", name)
+        unit = field.unit
+        kwargs.setdefault("stitle", f"{name} ({unit})")
```

Comment:
@PProfizi shouldn't you use the kwargs = self._set_scalar_bar_title(kwargs) method ?